sqalx_test.go

```diff
@@ -35,6 +37,7 @@ func TestSqalxConnectPostgreSQL(t *testing.T) {
 func TestSqalxConnectMySQL(t *testing.T) {
 	dataSource := os.Getenv("MYSQL_DATASOURCE")
 	if dataSource == "" {
+		fmt.Println("DEBUGDEBUG - skipping due to empty MYSQL_DATASOURCE")
```

Comment:
```suggestion
		t.Log("DEBUGDEBUG - skipping due to empty MYSQL_DATASOURCE")
```

Reply:
I didn't want to commit this :see_no_evil: 
Cleaned up the text and switched to `t.log` :ok_hand: